app/controllers/api/ads_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Api::AdsController < ApplicationController
+  include Rails::Pagination # https://github.com/davidcelis/api-pagination
+
+  MIN_PER_PAGE = 20
+  MAX_PER_PAGE = 100
+
+  def index # rubocop:disable Metrics/AbcSize
+    @ads = Ad
+      .active
+      .strict_loading
+      .includes(:city)
+      .ordered
+      .paginate(page: params[:page])
+      .for_kind(params[:kind])
+      .for_category(params[:category])
+      .for_city(params[:city_id])
```

Comment:
Is the intention that the api pages mirror the web interface pages? That seems like a reasonable expectation so If that's also your expectation then I'd DRY this into a method on the Ad so API and Web data doesn't accidentally diverge.

Reply:
Great suggestion. Implemented in https://github.com/vlado/earthquake-croatia/pull/131/commits/d022c078cdabe57c933f48a22c5e074e258c8b7f

Couldn't think of better name then `.for_index_page`, suggestions welcome :)